utils/global-state-update-gen/src/generic/state_tracker.rs

```diff
@@ -220,6 +220,20 @@ impl<T: StateReader> StateTracker<T> {
 
     /// Sets the bid for the given account.
     pub fn set_bid(&mut self, public_key: PublicKey, bid: Bid, slash: bool) {
+        let account_hash = public_key.to_account_hash();
+        let preexisting_unbonds = self.reader.get_unbonds();
+        let preexisting_unbonds = preexisting_unbonds.get(&account_hash);
```

Comment:
I'm guessing the borrow checker didn't allow for doing it in a single line?

Reply:
Yes, `get_unbonds` returns an owned (temporary) data, so I cannot return a reference to one of its items with `get`.

Reply:
I added `.cloned()`, I think it's indeed more readable.